pettingzoo/atari/base_atari_env.py

```diff
@@ -171,7 +171,9 @@ def __init__(
 
     def _seed(self, seed=None):
         if seed is None:
-            _, seed = seeding.np_random()
+            self.np_random, seed = seeding.np_random()
+        else:
+            self.np_random, seed = seeding.np_random(seed)
```

Comment:
You also need to fix line 182-183 of this file. Possibly remove the conditional and just do:
```py
self._seed(seed=seed)
```

Reply:
Oh nvm it looks like this is related to conversions.py, you mentioned you hadn't dived into that yet. Trying to think of what the best way to handle that is. I guess internally in there maybe we can just hard code adding code like `self.np_random = self.env.unwrapped.np_random` or something, but not sure if there's other things which are broken like this as well

Reply:
Argh, I think it is because the conversion wrappers do not give access to the attributes. It's much trickier to add this there since the API changes. I think the PR on supersuit side fixes this.